src/test/statistics/chunk_statistics/range_filter_test.cpp

```diff
@@ -196,98 +162,119 @@ TYPED_TEST(RangeFilterTest, CanPruneOnBounds) {
+  EXPECT_TRUE(filter->can_prune(PredicateCondition::Between, TypeParam{-999}, TypeParam{1}));
+  EXPECT_TRUE(filter->can_prune(PredicateCondition::Between, TypeParam{104}, TypeParam{1004}));
+  EXPECT_TRUE(filter->can_prune(PredicateCondition::Between, TypeParam{10'000'000}, TypeParam{20'000'000}));
 
-  EXPECT_FALSE(filter->can_prune(PredicateCondition::Between, {-3000}, {-500}));
-  EXPECT_FALSE(filter->can_prune(PredicateCondition::Between, {101}, {103}));
-  EXPECT_FALSE(filter->can_prune(PredicateCondition::Between, {102}, {1004}));
+  EXPECT_FALSE(filter->can_prune(PredicateCondition::Between, TypeParam{-3000}, TypeParam{-500}));
+  EXPECT_FALSE(filter->can_prune(PredicateCondition::Between, TypeParam{101}, TypeParam{103}));
+  EXPECT_FALSE(filter->can_prune(PredicateCondition::Between, TypeParam{102}, TypeParam{1004}));
 
   // SQL's between is inclusive
-  EXPECT_FALSE(filter->can_prune(PredicateCondition::Between, {103}, {123456}));
+  EXPECT_FALSE(filter->can_prune(PredicateCondition::Between, TypeParam{103}, TypeParam{123456}));
+
+  // TODO(bensk1): as soon as non-inclusive between predicates are implemented, testing
+  // a non-inclusive between with the bounds exactly on the value bounds would be humongous:
+  //  EXPECT_TRUE(filter->can_prune(PredicateCondition::BetweenNONINCLUSIVE, TypeParam{103}, TypeParam{123456}));
 }
 
 // Test larger value ranges.
 TYPED_TEST(RangeFilterTest, LargeValueRange) {
-  std::random_device rd;
-  auto rng = std::mt19937(rd());
-
-  // values on which is the range filter is later built on
-  pmr_vector<TypeParam> values;
-  values.reserve(1'000);
-
-  // Multiplying by 0.6 guarantees that the range filter will always yield a gap range
-  // right in the middle around value zero. Note, does not apply for unsigned values.
-  const TypeParam lower_bound_low_range = static_cast<TypeParam>(0.4 * std::numeric_limits<TypeParam>::lowest());
-  const TypeParam upper_bound_low_range = static_cast<TypeParam>(0.6 * lower_bound_low_range);
-  const TypeParam upper_bound_high_range = static_cast<TypeParam>(0.4 * std::numeric_limits<TypeParam>::max());
-  const TypeParam lower_bound_high_range = static_cast<TypeParam>(0.6 * upper_bound_high_range);
-
-  // We randomly create values between min_value(TypeParam) to 0.6*min_value(TypeParam) and
-  // 0.6*max(TypeParam) to max(TypeParam). Any value in between should be prunable.
-  distribution<TypeParam> lower_range_distribution(lower_bound_low_range, upper_bound_low_range);
-  distribution<TypeParam> high_range_distribution(lower_bound_high_range, upper_bound_high_range);
-  for (auto i = size_t{0}; i < 1'000; ++i) {
-    values.push_back(get_random_number<TypeParam>(rng, lower_range_distribution));
-    values.push_back(get_random_number<TypeParam>(rng, high_range_distribution));
-  }
+  const pmr_vector<TypeParam> values{static_cast<TypeParam>(0.4 * std::numeric_limits<TypeParam>::lowest()),
```

Comment:
Can you extract std::numeric_limits<TypeParam>::lowest() and std::numeric_limits<TypeParam>::max() into variables for readability